parity.gemspec

```diff
@@ -9,7 +9,6 @@ Gem::Specification.new do |spec|
   eos
 
   spec.email = ["dan@thoughtbot.com"]
-  spec.executables = ["development", "staging", "production"]
```

Comment:
Awesome. My intent is to test this with a beta Homebrew formula update before merging (which will be a first for me).